composer.json

```diff
@@ -65,7 +65,7 @@
         "magento/composer": "1.6.0",
         "magento/magento-composer-installer": ">=0.1.11",
         "magento/zendframework1": "~1.14.2",
-        "monolog/monolog": "^1.17",
+        "monolog/monolog": "~1.26.1",
```

Comment:
@konarshankar07 Thanks for the contribution! Can the library be updated by updating the composer.lock file only?